DuckDuckGo/TabViewController.swift

```diff
@@ -2387,10 +2387,12 @@ extension TabViewController: SecureVaultManagerDelegate {
         }
     }
 
+    // swiftlint:disable function_parameter_count
     func secureVaultManager(_: SecureVaultManager,
                             promptUserToAutofillCredentialsForDomain domain: String,
                             withAccounts accounts: [SecureVaultModels.WebsiteAccount],
                             withTrigger trigger: AutofillUserScript.GetTriggerType,
+                            onAccountSelected: @escaping (SecureVaultModels.WebsiteAccount?) -> Void,
```

Comment:
I had to make this one change on the iOS side to pass back to BSK that an account was selected & `lastUsed` needs to be updated as a result (regardless of whether or not a user successfully authenticates - as per Android rules spec)